### PR TITLE
Issue #30344 in file logging/config/cluster-logging-curator.adoc

### PR DESCRIPTION
**Pull request by rolfedh**
- For versions 4.5+
- Issue #30344
- Direct link to doc preview: https://deploy-preview-30429--osdocs.netlify.app/openshift-enterprise/latest/logging/config/cluster-logging-curator.html
- Ready for peer review and merge

### CODE REVIEW

**Comment by bmcelvee** [comment]
LGTM!

File: modules/cluster-logging-about-curator.adoc [hunk 1]
```diff
@@ -5,5 +5,4 @@
 [id="cluster-logging-about-curator_{context}"]
 = About logging curation
```
[the remark this answers is not in the document]
Comment (rolfedh) [reply modules/cluster-logging-about-curator.adoc:6]: I checked with @mburke5678. It was intentionally removed from the assemblies a while back.

**Pull request merged**